spec/models/transformation_mapping_spec.rb

```diff
@@ -159,19 +164,24 @@
     end
 
     context 'without VM list' do
+      let!(:this_vm) { FactoryBot.create(:vm_vmware, :name => 'this_test_vm', :ems_cluster => src, :ext_management_system => FactoryBot.create(:ext_management_system)) }
```

Comment:
Please don't specify `:name` as it is sequenced by the factory

Reply:
I can remove it.  I added them for clarity during debugging.

Reply:
specifying this name for verify in the spec.

Reply:
will modify it when implementing lan_validation